tests/Makefile

```diff
@@ -152,20 +152,25 @@ check: $(BINARIES)
 
 test: check
 
+prepare:
+	echo "  Prep  ../TESTS"
+	mkdir -p "../TESTS/host_tests"
```

Comment:
@AndrzejKurek , how and where this TEST folder was created previously ?

Reply:
I would assume that it was created in python test generating scripts (and we might not have to create the folder ourselves if we run them first), but it is better to have it explicitly created to note that we need additional preparation - like the script that gets copied.